.github/CODEOWNERS

```diff
@@ -972,10 +972,10 @@
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
 # PRLabel: %Oracle
-/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs
+/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs @v-eelhomsi
```

Comment:
v-eelhomsi requires public Microsoft and Azure memberships.   But, hey - at least they exist now!

Reply:
Not only do I now exist - I'm officially public!

Reply:
Confirmed!  You're real and official.